testcases/misc_tests/p11sak_test.sh

```diff
@@ -297,6 +297,16 @@ p11sak import-key ec private --slot $SLOT --pin $PKCS11_USER_PIN --label "import
 RC_P11SAK_IMPORT=$((RC_P11SAK_IMPORT + $?))
 p11sak import-key ec public --slot $SLOT --pin $PKCS11_USER_PIN --label "import-ec-secp521r1-public" --file $DIR/ec-key-secp521r1.pem --attr sX
 RC_P11SAK_IMPORT=$((RC_P11SAK_IMPORT + $?))
+# edwards/montgomery
+if [[ -n $(openssl version | grep "OpenSSL 3.") && -n $( pkcsconf -t -c $SLOT | grep "Model: EP11") ]]; then
+	p11sak import-key ec private --slot $SLOT --pin $PKCS11_USER_PIN --label "import-ec-ed25519-private" --file $DIR/ed25519-private-key.pem --attr sX
+	RC_P11SAK_IMPORT=$((RC_P11SAK_IMPORT + $?))
+	p11sak import-key ec public --slot $SLOT --pin $PKCS11_USER_PIN --label "import-ec-ed25519-public" --file $DIR/ed25519-public-key.pem --attr sX
+	RC_P11SAK_IMPORT=$((RC_P11SAK_IMPORT + $?))
+else
+	echo "Skip importing edwards/montgomery keys, OpenSSL version not supporting it or not EP11 token"
```

Comment:
Do you intentionally leave out tests for ed448? 
Also not sure if there's a difference between ed448 and x448 in the created key material.

Reply:
Yes, I decided to test only one out of the 4 possibilities with p11sak:

- ed25519
- ed448
- x25519
- x448

It all runs through the same code in p11sak, only within OpenSSL it may make a difference. 
To test the OCK token support of those 4 curves, we already have tests in testcases_crypto/ec_tests.c.